_posts/2017/2017-05-09-how-to-successfully-run-a-tech-blog.md

```diff
@@ -0,0 +1,94 @@
+For our ePages DevBlog we soon established the following topic categories:
+
+* Tech Stories
+* Working at ePages
+* Events
+* API
+* Agile
+
+With these categories in mind, we then developed an editorial calendar to determine what colleague would cover which topic area for which date.
+Usually, we have a rough content plan for the next two months and try to publish at least four times per months.
+We soon created a rotating system to make sure that our different developer teams take turns in writing blog posts.
+
+## Define guidelines for tech blog contributors
+
+Once you've determined the new blog's content structure, it is time to start with the creative writing process.
+For a company tech blog there will probably be more than only one contributor.
+Therefore, it is essential to define some basic guidelines for the blog post writers.
+At ePages, there are many developers, scrum masters and other colleagues from our R&D department that deliver great blog post ideas and content.
+So, for everyone to follow a similar writing and formatting pattern, we defined some blog post guidelines.
+These guidelines determine issues concerning the markdown style, including layout, headings, images, code quotation rules, SEO optimization, and alike.
+
+There is two essential elements that I would like to point out concerning content creation:
+Firstly: give it a personal touch!
+There is nothing worse than reading a post without a little wit and charm.
+Secondly, refer to other techies and relevant blogs!
+This does not only enhance your blog's credibility, but also gives you a better rating regarding search rankings.
+
+## Tools for tracking blog performance
+
+Now, after proof-reading your blog post and running the final tests, it is ready for publishing.
+There a some very helpful tools that can analyze your blog performance.
+For our ePages DevBlog, we signed up for [Hotjar](https://www.hotjar.com/) as well as connected our blog to [Google Analytics](https://analytics.google.com).
+Every time a new post is published, we create a heatmap at the analytics tool Hotjar in order to track visitor numbers and to visualize reader's bounce rates.
+Using Hotjar will let you learn about reading patterns and probably that readers are lazy and prefer shorter blog posts over long ones {% emoji wink %}.
+Google Analytics, on the other hand, is a very powerful instrument when it comes to measuring [blog traffic](https://www.blogmutt.com/blog/measuring-blog-traffic-with-google-analytics) and analyzing your blog's performance.
+
+## Sharing is caring: the Twitter magic
+
+Every blog post we publish, we also share on our [ePagesDevs Twitter](https://twitter.com/epagesdevs) channel and encourage our colleagues and followers to engage in the sharing fun.
+For broadening our readership, we not only tweet about our latest blog post publications but also promote articles from the archives, publish our job openings or retweet other relevant content for our followers.
```

Comment:
To increase our readership we...
Link "job openings" with "we're hiring" section.